02a-allow-all-traffic-to-an-application.md

```diff
@@ -12,8 +12,7 @@ void, and allow all traffic to it from its namespace and other namespaces.
 
 Start a `web` application:
 
-    kubectl run --generator=run-pod/v1 web --image=nginx \
-        --labels=app=web --expose --port 80
+    kubectl run web --image=nginx --labels=app=web --expose --port 80
```

Comment:
Replace --labels=app=web with --labels="app=web"

Reply:
Done, also changed `port 80` to `port=80`